### PR TITLE
Tagger tests

### PR DESCRIPTION
**Pull request by pastika**

### CODE REVIEW

File: python/processors/Stop0l_postproc.py [hunk 1]
```diff
@@ -59,16 +129,32 @@ def main(args):
...
             # statusFlag 0x2100 corresponds to "isLastCopy and fromHardProcess"
             # statusFlag 0x2080 corresponds to "IsLastCopy and isHardProcess"
-            GenPartFilter(statusFlags = [0x2100, 0x2080]),
+            GenPartFilter(statusFlags = [0x2100, 0x2080, 0x2000], pdgIds = [0, 0, 22], statuses = [0, 0, 1]),
+            TopTaggerProducer(recalculateFromRawInputs=True,                   AK4JetInputs=("Jet_pt",              "Jet_eta", "Jet_phi", "Jet_mass"),              topDiscCut=0.6),
+            TopTaggerProducer(recalculateFromRawInputs=True, suffix="JESUp",   AK4JetInputs=("Jet_pt_jesTotalUp",   "Jet_eta", "Jet_phi", "Jet_mass_jesTotalUp"),   topDiscCut=0.6),
+            TopTaggerProducer(recalculateFromRawInputs=True, suffix="JESDown", AK4JetInputs=("Jet_pt_jesTotalDown", "Jet_eta", "Jet_phi", "Jet_mass_jesTotalDown"), topDiscCut=0.6),
+        ]
+    else:
+        if DataDepInputs["Data"][args.era + args.dataEra]["redoJEC"]:
```
Comment (zhenbinwu) [inline python/processors/Stop0l_postproc.py]: so the dataEra will be just B/C/D/E ...?  Is this supported from the current cfg file?

File: python/modules/BtagSFWeightProducer.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        self.bTagEffFile = bTagEffFile
+        self.sampleName = sampleName
+        self.fileDirectory = fileDirectory
+
+    def beginJob(self):
+        ROOT.TH1.AddDirectory(False)
+        
+        fin = ROOT.TFile.Open(self.fileDirectory + "/" + self.bTagEffFile)
+
+        self.h_eff_b          = fin.Get(("n_eff_b_" + self.sampleName));
```
Comment (zhenbinwu) [inline python/modules/BtagSFWeightProducer.py]: You might run into samples without the pregenerated n_eff in the root file. Is there a fallback or error control?

Comment (pastika) [reply python/modules/BtagSFWeightProducer.py]: Thanks for the reminder, I had meant to put in such a safety, but obviously didn't.

File: python/processors/Stop0l_postproc.py (hunk 1 above)
Comment (pastika) [reply python/processors/Stop0l_postproc.py]: This is something we need to discuss with Scarlet and Anna.  Last night I learned the data names are not uniformly formatted between 2016/17/18 in the cfg files.  Once this is fixed so they are all separated by period, we should have no issue.

File: python/modules/BtagSFWeightProducer.py (hunk 2 above)
Comment (pastika) [reply python/modules/BtagSFWeightProducer.py]: I pushed another commit to address this.

File: python/processors/Stop0l_postproc.py (hunk 1 above)
Comment (zhenbinwu) [reply python/processors/Stop0l_postproc.py]: Ok, I asked @ahenckel to talk with you to figure out what you need and how we can propagate from cfg file via SubmitLPC.py to here.

**Comment by zhenbinwu** [comment]
I am intend to start merging this PR so that we can start the integration. Further optimization on this PR will be added later on

**Pull request merged**